gen/nvml/init.go

```diff
@@ -93,6 +93,10 @@ var nvmlDeviceGetAttributes = nvmlDeviceGetAttributes_v1
 var nvmlComputeInstanceGetInfo = nvmlComputeInstanceGetInfo_v1
 var nvmlDeviceGetComputeRunningProcesses = nvmlDeviceGetComputeRunningProcesses_v1
 var nvmlDeviceGetGraphicsRunningProcesses = nvmlDeviceGetGraphicsRunningProcesses_v1
+var GetBlacklistDeviceCount = GetExcludedDeviceCount
```

Comment:
Just to be clear, this is just a rename to allow this to be backward compatible?